server/serverless.yml

```diff
@@ -1,16 +1,28 @@
 service: sls-cms-server-app
 
+custom:
+    env_src: ${opt:env_src, 'default'} # pass a CLI flag to serverless commands to change environment var source
+    # i.e. `sls deploy --env_src=testing` 
+
 provider:
+  stage: ${env:DEV_NAME, 'dev'} # if DEV_NAME is set it will deploy to that stage, defaults to stage=dev
```

Comment:
so multiple devs can have isolated sandbox environments within an account
serverless will append this string to all resources